packages/node_modules/@node-red/runtime/lib/flows/Subflow.js

```diff
@@ -43,6 +43,54 @@ function evaluateInputValue(value, type, node) {
     return redUtil.evaluateNodeProperty(value, type, node, null, null);
 }
 
+/*!
+ * Get value of environment variable defined in group node.
+ * @param {String} group - group node
+ * @param {String} name - name of variable
+ * @return {Object} object containing the value in val property or null if not defined
+ */
+function getGroupSetting(node, group, flow, name) {
```

Comment:
Why is this defined in both `Subflow.js` and `util.js`? I can see some slight differences in the code, but it isn't clear why this function has to be duplicated for subflows.

Reply:
The cause is that the processing of the original environment variables is divided.   Since it is related to the implementation of the following credentials, I have integrated into one.